web_developer_mode_optional/static/description/icon.svg

```diff
@@ -0,0 +1,184 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Ah, sorry, browser doesn't show me the preview when I review it and I think it was the template one. Then I totally agree with Alejandro that the SVG must be there as en editable source. It's the same as if you include the compiled python pyc, but not the py :wink: